certbot/certbot/_internal/cli/verb_help.py

```diff
@@ -102,6 +102,11 @@
         "opts": 'Options useful for the "show_account" subcommand:',
         "usage": "\n\n  certbot show_account [options]\n\n"
     }),
+    ("reconfigure", {
+        "short": "Update configuration information for a certificate specified by --cert-name",
```

Comment:
Do we call this "information" anywhere else? Maybe "renewal configuration" or just "configuration"?

```suggestion
        "short": "Update configuration for a certificate specified by --cert-name",
```